### PR TITLE
Make code samples more consistent

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

**Review by njooma — COMMENTED** [review]
All the input names are inconsistent -- you have `machine`, `robot`, and `myRobotWithController`. Should they all just be `machine`?

**Review by stuqdog — APPROVED** [review]
lgtm modulo naveed's comment.

File: src/viam/components/input/input.py [hunk 1]
```diff
@@ -274,6 +274,8 @@ async def trigger_event(
         """Directly send an Event (such as a button press) from external code
 
         ::
+            my_controller = Controller.from_robot(
+                robot=myRobotWithController, name="my_controller")
```
Comment (npentrel) [inline src/viam/components/input/input.py:278]:
```suggestion
                robot=machine, name="my_controller")
```

File: src/viam/services/motion/motion.py [hunk 2]
```diff
@@ -238,6 +238,8 @@ async def stop_plan(
 
         ::
 
+            motion = MotionClient.from_robot(robot=robot, name="builtin")
```
Comment (npentrel) [inline src/viam/services/motion/motion.py:241]:
```suggestion
            motion = MotionClient.from_robot(robot=machine, name="builtin")
```

**Comment by npentrel** [comment]
Good catch - thanks

**Pull request merged**